composer.json

```diff
@@ -16,7 +16,7 @@
       "illuminate/http":"5.5.x|5.6.x|5.7.x",
       "illuminate/routing":"5.5.x|5.6.x|5.7.x",
       "illuminate/support":"5.5.x|5.6.x|5.7.x",
-      "philkra/elastic-apm-php-agent":"6.*.*",
+      "philkra/elastic-apm-php-agent":"dev-48-allow-explicit-timer-start",
```

Comment:
I know this is just for testing, but please add min version `6.5.3` of the agent, to ensure the client is actually able to perform what it should.